src/Triangle.java

```diff
@@ -0,0 +1,67 @@
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon {
+
+	public Triangle(int numOfSides, ArrayList<Point> coors) { // 2 i
```

Comment:
u don't need the `int numOfSides` parameter since we all know the side numbers of a triangle "3".